pkgs/top-level/perl-packages.nix

```diff
@@ -21802,7 +21802,10 @@ let
       description = "Interface to read and parse BibTeX files";
       license = with lib.licenses; [ artistic1 gpl1Plus ];
     };
-  };
+  } // lib.optionalAttrs stdenv.isAarch64 {
+    # libbtparse.so: cannot open shared object file (aarch64 only)
+    patches = [ ../development/perl-modules/TextBibTeX-use-lib-on-aarch64.patch ];
+  });
```

Comment:
Please do not apply this patch conditionally. It does not break amd64 and would only bit rot over time.

Reply:
very good point